soh/soh/z_message_OTR.cpp

```diff
@@ -154,4 +154,9 @@ extern "C" void OTRMessage_Init()
         CustomMessage("You look bored. Wanna go out for a&walk?\x1B&%gYes&No%w",
                       "Du siehst gelangweilt aus.&Willst du einen Spaziergang machen?\x1B&%gJa&Nein%w",
                       "Tu as l'air de t'ennuyer. Tu veux&aller faire un tour?\x1B&%gOui&Non%w"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_FISHERMAN_LEAVE,
+        CustomMessage("Hey! Hey!&You can't take the rod out of here!^I'm serious! Do you want to&quit?\x1B&%gYes&No%w",
```

Comment:
The "I'm serious!" here seems a bit redundant, he doesn't say it that often.

Reply:
That one's sort of vanilla: "Hey! Hey! You can't take the rod out of here! When you quit fishing, return the rod to the counter. I'm serious!" Happy to cut it, though, since the flow is a bit different.

Reply:
Ah, I see what you mean. Maybe leave it on the first page there instead of on the "Do you want to quit?" box?